cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/CpuCapacity.java

```diff
@@ -36,4 +36,13 @@ protected JsonObject getJson() {
     public String toString() {
         return this.getJson().toString();
     }
+
+    /**
+     * Retrieves the value of the 'cores' property.
+     *
+     * @return The value of the 'cores' property.
+     */
+    public String getCores() {
+        return cores;
+    }
```

Comment:
The usual question -> oes it have to be public? The classes using it seem to be in the same package, so `/* test */` might be sufficient?

Reply:
As CruiseControlTest is in a different package to this, it needs to be public to access it from CruiseControlTest.